### DISCUSSION TITLE
AMD APU?

### DISCUSSION THREAD

**Discussion by ignacio82** [blakeblackshear/frigate#9180] [Ask A Question]
I'm running frigate using docker on a Ryzen 9 6900HX(Up to 4.9GHz), Radeon 680M Graphics. 

But it seems GPU acceleration does not work:

![image](https://github.com/blakeblackshear/frigate/assets/1833309/90701849-3858-4022-81c6-6f530506ccc5)

docker-compoese:

```
version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    image: ghcr.io/blakeblackshear/frigate:stable
.
.
.
    devices:
      - /dev/bus/usb:/dev/bus/usb # passes the USB Coral, needs to be modified for other versions
      - /dev/dri/renderD128:/dev/dri/renderD128
      - /dev/kfd:/dev/kfd
      - /dev/dri/card0:/dev/dri/card0
.
.
.
    environment:
      - LIBVA_DRIVER_NAME:radeonsi      
```
 
Config.yaml:

```
ffmpeg:
  hwaccel_args: preset-vaapi    
```
    

Log:
```
2024-01-02 17:03:56.920673339  [2024-01-02 17:03:55] frigate.util                   ERROR   : Unable to poll intel GPU stats: No device filter specified and no discrete/integrated i915 devices found
```

What am I missing?

**Comment by NickM-27** [comment]
docker compose variables have this format:

```
    environment:
      LIBVA_DRIVER_NAME: "radeonsi"
```

**Reply by andrico21** [reply]
Both accepted, it's mentioned in Docker documentation.

Upd: well spotted, @NickM-27 - thanks.

**Reply by NickM-27** [reply]
The other format is `-LIBVA_DRIVER_NAME=radeonsi` the format OP used is not accepted or in the docker docs https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/
